test/e2e/specs/jetpack/plugins__install-remove.ts

```diff
@@ -21,31 +23,33 @@ describe( DataHelper.createSuiteTitle( 'Jetpack: Plugin' ), function () {
 	let pluginsPage: PluginsPage;
 	let noticeComponent: NoticeComponent;
 	let siteURL: string;
+	let restAPIClient: RestAPIClient;
 
 	beforeAll( async function () {
+		// Ensure known good state by removing the plugin if already installed.
+		restAPIClient = new RestAPIClient( SecretsManager.secrets.testAccounts.jetpackUser );
+		const siteID = SecretsManager.secrets.testAccounts.jetpackUser.testSites?.primary.id as number;
+		const response = await restAPIClient.removePluginIfInstalled( siteID, pluginName );
+
+		if ( response ) {
+			console.log( `Successfully removed the plugin '${ pluginName }'.` );
+		} else {
+			console.warn( `Failed to remove the plugin '${ pluginName }'; no action performed.` );
```

Comment:
**nit/non-blocking**: I suppose it doesn't really matter for this spec since the plugin is always installed prior to the spec running, but I do wonder if the message should be a little less "alarming" for lack of a better word.

I say this because it is possible to have a test site not have the plugins installed (eg. for a new test site, or someone has cleaned it up manually). In those cases the response would be `null` and thus the `else` codepath will be taken, which prints the failure message when in fact it isn't so much a failure as "no action was required".

Reply:
Thank you for this! I updated the message and used `console.log` 👍🏻 fixed in [f75144c](https://github.com/Automattic/wp-calypso/pull/71082/commits/f75144cf929db1551eb281b9dfb874dabb3b59cc).